unit_tests/test_version_utils.py

```diff
@@ -761,30 +761,33 @@ def test_get_branched_repo(scylla_version, distro, expected_repo):
 
 
 @pytest.mark.parametrize(
-    "version, expected_repo, input_repo",
+    "version, expected_repo",
     (
-        ("6.2.2", "scylladb/scylla", None),
-        ("6.2.3", "scylladb/scylla", None),
-        ("6.2.4", "scylladb/scylla", None),
-        ("6.2.66", "scylladb/scylla", None),
-        ("2024.1.1", "scylladb/scylla-enterprise", None),
-        ("2024.2.13", "scylladb/scylla-enterprise", None),
-        ("2024.2.14", "scylladb/scylla-enterprise", None),
-        ("enterprise:latest", "scylladb/scylla-enterprise", None),
-        ("branch-2024.2:latest", "scylladb/scylla-enterprise", None),
-        ("branch-2024.1:latest", "scylladb/scylla-enterprise", None),
-        ("2025.1.0", "scylladb/scylla", None),
-        ("2025.2.99", "scylladb/scylla", None),
-        ("2025.4.0", "scylladb/scylla", None),
-        ("branch-2025.1:latest", "scylladb/scylla", None),
-        ("branch-2025.2:latest", "scylladb/scylla", None),
-        ("branch-2025.3:latest", "scylladb/scylla", None),
-        ("branch-2025.4:latest", "scylladb/scylla", None),
-        ("master:latest", "scylladb/scylla", None),
-        ("2026.1.0-dev-0.20251217.55f4a2b75472", "scylladb/scylla-nightly", None),
-        ("2026.1.0-dev-0.20251217.55f4a2b75472", "scylladb/scylla-nightly", "scylladb/scylla-nightly"),
-        ("2026.1.0.20251217.55f4a2b75472", "scylladb/scylla-nightly", "scylladb/scylla-nightly"),
+        ("6.2.2", "scylladb/scylla"),
+        ("6.2.3", "scylladb/scylla"),
+        ("6.2.4", "scylladb/scylla"),
+        ("6.2.66", "scylladb/scylla"),
+        ("2024.1.1", "scylladb/scylla-enterprise"),
+        ("2024.2.13", "scylladb/scylla-enterprise"),
+        ("2024.2.14", "scylladb/scylla-enterprise"),
+        ("enterprise:latest", "scylladb/scylla-enterprise"),
+        ("branch-2024.2:latest", "scylladb/scylla-enterprise"),
+        ("branch-2024.1:latest", "scylladb/scylla-enterprise"),
+        ("2024.5.0-dev-0.20251217.55f4a2b75472", "scylladb/scylla-enterprise-nightly"),
+        ("2024.99.99-weirdnonsemver", "scylladb/scylla-enterprise-nightly"),
+        ("2025.0.0-dev-0.20251217.55f4a2b75472", "scylladb/scylla-enterprise-nightly"),
```

Comment:
This is wrong any 2025 and up shouldn't be from enterprise

Reply:
And kind of pointless to test non valid versions, i.e. this won't exist in any repo...